core/src/main/java/com/linecorp/armeria/server/RouteBuilder.java

```diff
@@ -413,6 +415,15 @@ RouteBuilder matchesHeaders(List<RoutingPredicate<HttpHeaders>> headerPredicates
         return this;
     }
 
+    /**
+     * Sets whether the {@link Route} would set a deferred exception to a {@link RoutingContext} instance.
+     * Currently, it is used only when configuring {@link Route}s to fallback services.
+     */
+    RouteBuilder canSetDeferredException(boolean setDeferredException) {
```

Comment:
Suggestion: `allowDeferredException(boolean allowDeferredException)`?

Reply:
It'd be better name. Thank you, professor! 
Will change the method name as well.

Reply:
I'm happy if the suggestion was good. 

> Thank you, professor!

🤣